x-pack/legacy/plugins/transform/public/app/sections/create_transform/components/source_index_preview/use_source_index_data.ts

```diff
@@ -48,23 +36,34 @@ const isErrorResponse = (arg: any): arg is ErrorResponse => {
   return arg.error !== undefined;
 };
 
-type SourceIndexSearchResponse = ErrorResponse | SearchResponse<any>;
+// The types specified in `@types/elasticsearch` are out of date and still have `total: number`.
+interface SearchResponse7 extends SearchResponse<any> {
```

Comment:
why the 7 at the end of the name?

Reply:
The `@types/elasticsearch` types are quite out of date (6.something) so I put the `7` there as a fix saying for `7.x`. Hope this is temporary and at some point we get updated types.